### ISSUE TITLE
Add in some basic minetype for Mobalytics so browser can redirect properly

### ISSUE THREAD

**Issue by spstarr** [sekwah41/mobalytics-repackager#23]
I sorta got a MIMEtype setup

```
[Desktop Entry]
Name=Mobalytics
StartupWMClass=mobalytics-desktop
Comment=Mobalytics Desktop
GenericName=Mobalytics Desktop for Linux
Exec=/home/spstarr/mobalytics/mobalytics-desktop %U
Type=Application
StartupNotify=true
MimeType=x-scheme-handler/mobalytics;
```

`xdg-mime default mobalytics.desktop x-scheme-handler/mobalytics`

Though once I run it unpackaged, login, then I can use the AppImage method of running the program.

**Comment by spstarr** [comment]
It is hooked into the desktop:

![image](https://user-images.githubusercontent.com/2081708/159417787-ffd6ac56-d178-495d-8e36-cb55b34e8ecc.png)

**Comment by ProPanek** [comment]
This should be added to the README.md tbh

> I sorta got a MIMEtype setup install the desktop file in ~/.local/share/applications
> 
> ```
> [Desktop Entry]
> Name=Mobalytics
> StartupWMClass=mobalytics-desktop
> Comment=Mobalytics Desktop
> GenericName=Mobalytics Desktop for Linux
> Exec=/home/spstarr/Mobalytics-1.98.239.AppImage %U
> Icon=/usr/share/pixmaps/mobalytics.png
> Type=Application
> StartupNotify=true
> MimeType=x-scheme-handler/mobalytics;
> ```
> 
>     * Extract the PNG icon from the asar resources
> 
> 
> `xdg-mime default mobalytics.desktop x-scheme-handler/mobalytics` `update-mime-database ~/.local/share/applications/`

**Comment by sekwah41** [comment]
Turns out the reason that this was no longer being set it electron-builder removing desktop integration as part of the tool in v21
![image](https://user-images.githubusercontent.com/3209834/227817284-e8b66448-dbe9-492f-8af1-001ab0526a08.png)

I will also add it to the readme, but i will revert the builder back down to v20 too.

**Comment by sekwah41** [comment]
Yup downgrading brings back this menu. I was wondering why it had vanished and though it was some weird manjaro update not electron builder.

Btw im back  to actively developing my projects so if you guys have any more issues or suggestions feel free to poke me.

**Comment by sekwah41** [comment]
Seems there is a reason behind it, though ill look to get some stuff working still https://github.com/electron-userland/electron-builder/releases/tag/v21.0.1
